Tests/MessageTests.swift

```diff
@@ -64,69 +61,63 @@ class MessageTests: XCTestCase {
 
     func testMessageForDay() {
-        let messageGenerator = MessageGenerator()
-
         let beginningOfDayString = "2014-07-15 01:00"
         let endOfDayString = "2014-07-15 23:40"
 
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "yyyy-MM-dd HH:mm"
 
-        let beginningOfDayDate = dateFormatter.date(from: beginningOfDayString)
-        let endOfDayDate = dateFormatter.date(from: endOfDayString)
+        let beginningOfDayDate = dateFormatter.date(from: beginningOfDayString)!
+        let endOfDayDate = dateFormatter.date(from: endOfDayString)!
 
-        let beginMessage = messageGenerator.generateMessage(date: beginningOfDayDate!, hemisphere: .southern, daylightLengthDifference: 100)
-        let endMessage = messageGenerator.generateMessage(date: endOfDayDate!, hemisphere: .southern, daylightLengthDifference: 100)
+        let beginMessage = Message(for: beginningOfDayDate, coordinates: Location.testLocation)
+        let endMessage = Message(for: endOfDayDate, coordinates: Location.testLocation)
 
         XCTAssertEqual(beginMessage.content, endMessage.content)
     }
 
     func testMessageForNight() {
-        let messageGenerator = MessageGenerator()
-
         let beginningOfDayString = "2014-07-15 01:00"
         let endOfDayString = "2014-07-15 23:40"
 
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "yyyy-MM-dd HH:mm"
 
-        let beginningOfDayDate = dateFormatter.date(from: beginningOfDayString)
-        let endOfDayDate = dateFormatter.date(from: endOfDayString)
+        let beginningOfDayDate = dateFormatter.date(from: beginningOfDayString)!
+        let endOfDayDate = dateFormatter.date(from: endOfDayString)!
 
-        let beginMessage = messageGenerator.generateMessage(date: beginningOfDayDate!, hemisphere: .southern, daylightLengthDifference: 100)
-        let endMessage = messageGenerator.generateMessage(date: endOfDayDate!, hemisphere: .southern, daylightLengthDifference: 100)
+        let beginMessage = Message(for: beginningOfDayDate, coordinates: Location.testLocation)
+        let endMessage = Message(for: endOfDayDate, coordinates: Location.testLocation)
 
         XCTAssertEqual(beginMessage.content, endMessage.content)
     }
 
     func testNotificationMessage() {
-        let messageGenerator = MessageGenerator()
-
-        let message = messageGenerator.messageForNotification(date: Date(), hemisphere: .southern, daylightLengthDifference: 100)
+        let message = Message.notificationMessage(for: Date(), coordinates: Location.testLocation)
 
         XCTAssertFalse(message.contains("**"))
     }
 
-    func testNotificationMessageAtNight() {
-        let nightString = "2014-07-15 00:00"
-
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm"
-        let nightDate = dateFormatter.date(from: nightString)
+    func testNightMessageOnlyAtNight() {
+        // 20st of December 2017 11:00 UTC
```

Comment:
`20th`

Reply:
😂 I changed the date and forgot to adjust that haha

Reply:
the twentiest, the most twenty of all.